Changelog.md

```diff
@@ -1,7 +1,8 @@
 # FOSSA CLI Changelog
 
-## Unreleased
+## v3.6.4
 
+- Fixes `--detect-vendored` on Windows. ([#1096](https://github.com/fossas/fossa-cli/pull/1096))
```

Comment:
[minor] let's add c/c++ prefix in front i.e. C/C++: Fixes `--detect-vendored` on Windows?